Source/LinqToDB/Data/DataReaderAsync.cs

```diff
@@ -206,6 +229,13 @@ public Task QueryForEachAsync<T>(T template, Action<T> action, CancellationToken
 			return QueryForEachAsync(action, cancellationToken);
 		}
 
+#if NATIVE_ASYNC
+		public IAsyncEnumerable<T> QueryForEachAsync<T>(T template, CancellationToken cancellationToken)
```

Comment:
```suggestion
		public IAsyncEnumerable<T> QueryForEachAsync<T>(T template, [EnumeratorCancellation] CancellationToken cancellationToken = default)
```

Is this the proper method name?  First, I'm not sure it should end in 'Async', and second, the other overloads here are overloads of the same method name without the template, correct?  So shouldn't this be named `QueryToAsyncEnumerable<T>(T template, ...)` ?

Reply:
Whoops. Missed one. Fixed in #732badc (will be uploaded when possible)